src/q5/tdg-INMA1691/exam/2020/Janvier/Majeure/tdg-INMA1691-exam-2020-Janvier-Majeure.tex

```diff
@@ -0,0 +1,83 @@
+\documentclass[fr]{../../../../../../eplexam}
+
+\hypertitle{}{5}{INMA}{1691}{2020}{Janvier}{Majeure}
```

Comment:
Tu as utilisé add.sh pour générer le fichier ? C'est bizarre que le titre ne soit pas automatiquement encodé.
```suggestion
\hypertitle{Théorie des graphes}{5}{INMA}{1691}{2020}{Janvier}{Majeure}
```

Reply:
Oui, mais il y a eut une erreur
`Traceback (most recent call last):
  File "course_link.py", line 11, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
`

Reply:
Ok, c'est que tu n'as pas yaml sur Python, pas grave =)